src/slack_notify.ml

```diff
@@ -42,11 +42,15 @@ let message =
   let doc = "Message to send" in
   Cmdliner.Arg.(required & pos 1 (some string) None & info [] ~docv:"MSG" ~doc)
 
+let attachment =
+  let doc = "Attachment text" in
+  Cmdliner.Arg.(value & opt (some string) None & info ["attachment"] ~docv:"MSG" ~doc)
+
 let info =
   let doc = "Posts messages to Slack" in
   Cmdliner.Term.info "slack-notify" ~doc
 
-let execute token username channel icon_url icon_emoji msg =
+let execute token username channel icon_url icon_emoji atxt msg =
```

Comment:
`attachment_text` would be better or something, since `atxt` is a rather unexpressive name.